misc.go

```diff
@@ -613,6 +613,7 @@ var tokenMap = map[string]int{
 	"TYPE":                     tp,
 	"UNBOUNDED":                unbounded,
 	"UNCOMMITTED":              uncommitted,
+	"UNICODE":                  unicodeType,
```

Comment:
should we?
```suggestion
	"UNICODE":                  unicode,
```

Reply:
After I do it, `make test` will fail:
```
...
./lexer.go:21: unicode redeclared in this block
        previous declaration at ./parser.go:635
...
./yy_parser.go:21: unicode redeclared as imported package name
        previous declaration at ./parser.go:635
...
```
"unicode" package can't be imported.

Reply:
how about we name it unicodeSym